tinyiiod.h

```diff
@@ -44,7 +47,9 @@ struct tinyiiod_ops {
 	int32_t (*open)(const char *device, size_t sample_size, uint32_t mask);
 	int32_t (*close)(const char *device);
 
-	ssize_t (*read_data)(const char *device, char *buf, size_t bytes_count);
+	ssize_t (*read_data)(const char *device, char *pbuf, size_t bytes_count);
```

Comment:
the `buf` -> `pbuf` rename is just noise; it doesn't need to change

Reply:
I will fix this